### ISSUE TITLE
No 'Access-Control-Allow-Origin' header is present for /socket.io/ URL

### ISSUE THREAD

**Issue by spidgorny** [AlexSergey/rockpack#13]
```
Access to XMLHttpRequest at 'http://localhost:62194/socket.io/?EIO=4&transport=polling&t=NU8nLXR' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```
This is a CRA project where I experimented with rockpack. Is this a bug or my fault?

**Comment by AlexSergey** [comment]
Thank you for your attention to the project! I've never faced this problem. Could you provide me please more details about your environment (OS, browser, rockpack version)?

**Comment by spidgorny** [comment]
OS: Windows 1909 (OS Build 18363.1316)
Browser: Chome Version 88.0.4324.150
Rockpack: 2.0.0-rc.5

**Comment by AlexSergey** [comment]
> OS: Windows 1909 (OS Build 18363.1316)
> Browser: Chome Version 88.0.4324.150
> Rockpack: 2.0.0-rc.5

Thank you for the details! I work on windows too but Rockpack works well in Windows, Mac, and Linux. I will try to figure out...

**Comment by AlexSergey** [comment]
@spidgorny ,

I don't understand what is going wrong.
What is http://localhost:62194/ ?
Is it your own logic in your application or you don't use socket in your example and it's a part of Rockpack's logic?

**Comment by AlexSergey** [comment]
@spidgorny 

Maybe I found a potential vulnerability. I published rockpack@2.0.0-rc.7. Please, use it and give me feedback because I can't reproduce it locally.

**Comment by spidgorny** [comment]
My app is not using WS at all. I believe it comes from the HMR. Does rockpack support HMR?
I'm trying to use rockpack on the existing project which was initially created with CRA.
If rockpack doesn't support HMR, it may explain the situation.
I've updated rockpack to rc8 and restarted - and wow - it works now.
(Another issue is that HMR refreshes the whole page, instead of doing in-place updates. Maybe I report it separately).
This ticket can be closed.
